site/docs/security/tls.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Encryption and Authentication using TLS
+prev: ../index.html
+next: ../sasl
+---
+
+Apache BookKeeper allows clients and autorecovery daemons to communicate over TLS, although this is not enabled by default.
+
+## Overview
+
+The bookies and clients need their own key and certificate in order to use TLS. The key is used for encryption, while the
```

Comment:
> clients need their own key and certificate

If you only want to do encryption, in general there's no need for the client to configure a certificate. A self-signed certificate should be created on the spot. (I have not checked if it's actually true in the current implementation)